rmgpy/rmg/model.py

```diff
@@ -830,9 +830,12 @@ def generate_thermo(self, spc, rename=False):
         if not spc.thermo:
             submit(spc, self.solvent_name)
 
+            # if rename and spc.thermo and spc.thermo.label != '':  # check if thermo libraries have a name for it
+            #     logging.info('Species {0} renamed {1} based on thermo library name'.format(spc.label, spc.thermo.label))
+            #     spc.label = spc.thermo.label
             if rename and spc.thermo and spc.thermo.label != '':  # check if thermo libraries have a name for it
```

Comment:
@lily90502 what did you and @mjohnson541 decide?

Reply:
This will only affect fragments' naming. Since we are using representative species for estimating thermo properties of fragments by replacing cutting label with a complicated structure. If the species match the one in thermo library, we will just use the values but still keep the species label the same as fragment's label.